tests/decoder_tests.rs

```diff
@@ -533,3 +534,18 @@ fn color_and_alpha_dimensions_do_not_match() {
     assert!(res.is_err());
     assert_avif_error!(res, DecodeAlphaFailed);
 }
+
+#[test]
+fn rgb_conversion_alpha_premultiply() -> AvifResult<()> {
+    let mut decoder = get_decoder("alpha.avif");
+    let res = decoder.parse();
+    assert!(res.is_ok());
+    let res = decoder.next_image();
+    assert!(res.is_ok());
+    let image = decoder.image();
+    let mut rgb = rgb::Image::create_from_yuv(image);
+    rgb.alpha_premultiplied = true;
```

Comment:
I suggest renaming the field to `premultiply_alpha` then.

Reply:
good idea. i will do this in a follow-up though.